collect_app/src/main/java/org/odk/collect/android/preferences/screens/MapsPreferencesFragment.kt

```diff
@@ -68,6 +68,13 @@ class MapsPreferencesFragment : BaseProjectPreferencesFragment(), Preference.OnP
         super.onSettingChanged(key)
         if (key == ProjectKeys.KEY_REFERENCE_LAYER) {
             findPreference<Preference>(ProjectKeys.KEY_REFERENCE_LAYER)!!.summary = getLayerName()
+        } else if (key == KEY_BASEMAP_SOURCE) {
```

Comment:
It looks like there's no testing for anything in `onSettingsChanged` here. No need to block this - I'll just create an issue as a follow-up.